src/tests/ftest/util/ServerUtils.py

```diff
@@ -33,18 +33,126 @@
 import signal
 import fcntl
 import errno
+import yaml
 
 from avocado.utils import genio
 
 sessions = {}
 
+DEFAULT_YAML_FILE = "utils/config/examples/daos_server_sockets.yml"
```

Comment:
I could see this not working at least the location when we have install packages in the future.  I guess I will see why we need these later on but I'm thinking it would be nice not to presume the test has access to these.

Reply:
In one of the further comment, we may be going to refer our own copy of input.yaml  from src/ftest/ so that might resolve issue in future in case some thing get changed for install packages. Thoughts?

Reply:
Base line yaml file created under ftest/data/daos_server_baseline.yml